apps/site/pages/en/learn/http/enterprise-network-configuration.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Enterprise Network Configuration
+layout: learn
+authors: Joyee Cheung
+---
+
+# Enterprise Network Configuration
+
+## Overview
+
+Enterprise environments often require applications to operate behind corporate proxies and use custom certificate authorities (CAs) for SSL/TLS validation. Node.js provides built-in support for these requirements through environment variables and command-line flags, eliminating the need for third-party proxy libraries in many cases.
+
+This guide covers how to configure Node.js applications to work in enterprise network environments:
```

Comment:
maybe add somewhere here "since node.js vX.X.X ofter support for proxy"

So user know witch version

Reply:
They are covered in the sections below (because different parts are supported from different versions, and are not rolled out together in one release).